packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -721,28 +723,19 @@ export default function ScheduleDetails({ modalProps, actions, id }) {
 
           <SimpleTransactionsTable
             renderEmpty={
-              state.transactionsMode === 'matched' &&
-              (() => (
-                <View
-                  style={{ padding: 20, color: colors.n4, textAlign: 'center' }}
-                >
-                  {state.error ? (
-                    <Text style={{ color: colors.r4 }}>
-                      Could not search: {state.error}
-                    </Text>
-                  ) : (
-                    'No transactions found'
-                  )}
-                </View>
-              ))
```

Comment:
Here's the bit for displaying an message when "Linked transactions" is selected.